src/components/notificationBanner/notificationBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link, Notification, Text } from "@nypl/design-system-react-components";
+
+const NotificationBanner = () => {
+  return (
+    <Notification
+      noMargin={true}
+      isCentered={true}
+      notificationContent={
+        <Text
+          sx={{
+            marginBottom: "0px",
+            fontWeight: "400",
+          }}
+        >
+          Digital Collections is getting a new look. Return to original
+          <Link href="https://digitalcollections.nypl.org/">
+            {" "}
```

Comment:
Move this to line 15 because the space is rendered as part of the link:
<img width="70" alt="Screen Shot 2023-12-13 at 10 40 39 AM" src="https://github.com/NYPL/digital-collections/assets/1280564/ccec5eed-d207-4222-88bf-220e0e5cdc3f">